### ISSUE TITLE
Bookmarklet works in Safari but not Brave

### ISSUE THREAD

**Issue by bwoodruff** [brave/browser-laptop#3218]
**Did you search for similar issues before submitting this one?**

Yes

**Describe the issue you encountered:**

Have the following bookmarklet:

`javascript:var%20text='##%20%5B'+document.title.replace('%20-%201Password%20Support','')+'%5D('+location.href+')';window.prompt(%22Copy%20to%20clipboard:%22,text);void(0);`

**Expected behavior:**

Bookmarklet should run the JavaScript which takes the page's title, strips "1Password Support" off it, and combines it with the page's URL into markdown. e.x. on this page:
https://support.1password.com/1password-com-getting-started/

It produces this prompt:
![get_to_know_1password_com_-_1password_support](https://cloud.githubusercontent.com/assets/553522/17722378/b481e57a-63ff-11e6-8853-62baa8e83573.png)

`## [Get to know 1Password.com](https://support.1password.com/1password-com-getting-started/)`

- Platform (Win7, 8, 10? macOS? Linux distro?):

![fullscreen_8_16_16__10_21_pm](https://cloud.githubusercontent.com/assets/553522/17722477/58a36638-6400-11e6-92bc-9cad47445695.png)

- Brave Version:

(see above)

- Steps to reproduce:
    1. Add this bookmarklet to Brave's bookmarks toolbar: `javascript:var%20text='##%20%5B'+document.title.replace('%20-%201Password%20Support','')+'%5D('+location.href+')';window.prompt(%22Copy%20to%20clipboard:%22,text);void(0);`
    2. Click the bookmarklet
    3. It should produce a JavaScript window.prompt() but it does not. It does nothing. Do the same in Safari and it works.

- Screenshot if needed:

- Any related issues:

**Comment by alexwykoff** [comment]
+1 from support : https://linkbubble.zendesk.com/agent/tickets/5237

**Comment by luixxiul** [comment]
https://community.brave.com/t/issues-with-bookmarklets/1999?u=suguru

**Comment by schultzter** [comment]
I think a lot of simple extensions are already available as generic bookmarklets. Fixing this, making bookmarklets work, would lower the priority or even eliminate a lot of Extension requests.

**Comment by hugobuddel** [comment]
Simple bookmarklets like "javascript:alert("hi");" work fine, at least on 0.18.36 on windows 10.

More complicated bookmarklets like those from pinboard work as well:
https://pinboard.in/howto/

The bookmarklet in this issue indeed does not work.

**Comment by benf-wspdigital** [comment]
@hugobuddel, thanks for testing.

With 0.18.36 on MacOS, the bookmarklets at https://pinboard.in/howto/ don't work. Attempting to add one as a bookmark results in a bookmark with the incorrect “about:blank” as the location.

**Comment by hugobuddel** [comment]
Yes you're right @benf-wspdigital, should have mentioned that. Creating bookmarklets is hard: you have to use another browser (or view source) to copy the link and then manually paste it in the bookmark, I've created #11088 for that. But the bookmarklet works once it has been made.

**Comment by hugobuddel** [comment]
It seems to still work fine in 0.22.22.

But note that pasting is easy part (in 0.22.22 at least). The hard part is to copy the link. You'd either need another browser to do that or use view-source. Trying to copy the link directly from the pinboard page will copy "about:blank", so that is what will be pasted.

**Comment by hugobuddel** [comment]
And choose 'copy', not 'copy link address' when using view-source.

**Comment by slmeyer** [comment]
OK - I see that magic somehow when using firefox to copy it works, though I do not understand why, thanks a lot

**Comment by hugobuddel** [comment]
You're welcome. Now if only it would be possible to access the bookmarklet easily, like a keyboard shortcut or so. Any ideas perhaps?

I have the 'bookmark toolbar' visible primarily for the pinboard bookmarklet. My laptop screen has a 16:9 aspect radio, so it takes quite some space. By now I've added some other bookmarks as well, so it is not so bad, but a better way to access bookmarklets would be appreciated.

**Comment by slmeyer** [comment]
I have added them to the bookmarks menu, so when those keyboard shortcuts are fixed - it'll work

**Comment by hugobuddel** [comment]
Indeed, that would do @slmeyer . Looked up the relevant issue for reference: #5212

**Comment by jf** [comment]
you know as somebody who's new to Brave, and is now experiencing this issue, it would be great to explain the `wontfix` label. I think I get it now after looking at https://github.com/brave/muon; but I think some brief explanation would help and be appreciated. But basically: the new version of Brave (when is that coming out?) should resolve this, so effort won't be put in to fix this for the current version.

@hugobuddel thank you for your helpful notes and tips here. I've found it better to simply use "Inspect Element" on the bookmarklet, and then double-click and copy the "href=" value.

**Comment by bsclifton** [comment]
@jf the new version is internally called `brave-core` and is available now on developer and beta channels. 😄 

You can grab a copy here:
https://brave.com/download-beta

**Comment by hugobuddel** [comment]
Thanks for the tip @jf, using Inspect Element is much nicer than what I've been doing. (It seems obvious now you mention it.) In brave-core it isn't a problem anymore though.
